PermissionsScraper/PermissionsScraper/Triggers/DescriptionsScraper.cs

```diff
@@ -69,7 +70,8 @@ public static void Run([TimerTrigger("0 0 9 * * 1-5")]TimerInfo myTimer, ILogger
                     return;
                 }
 
-                var servicePrincipalScopes = JsonConvert.SerializeObject(_scopesDescriptions, Formatting.Indented);
+                var servicePrincipalScopes = JsonConvert.SerializeObject(_scopesDescriptions, Formatting.Indented)
```

Comment:
Is this a file in a git repo? Then autocrlf setting can be configured at repository level to make sure that it is always checked in with `\n` for line endings instead of `\r\n`.

Reply:
Yes. It is a git repo file.
I will look into that option.